DEVELOPMENT.md

```diff
@@ -2,39 +2,45 @@
-
-Two ports will be open:
-
-- [localhost:8000](localhost:8000): the latest version of the application on the `phase-ii` branch
-- [localhost:9000](localhost:9000): live updating application (server reload for Django and HMR for Vue)
-
-### Common tasks
-
-- [`django-admin`](https://docs.djangoproject.com/en/4.1/ref/django-admin) commands:
-  - e.g. run migrations: `docker compose run --rm poetry run django-admin migrate`
-- `npm` commands and scripts (see [`scripts` in `package.json`](https://github.com/ResonantGeoData/RD-WATCH/blob/phase-ii/vue/package.json#L5)):
-  - e.g. lint and fix code: `docker compose run --rm npm run lint:fix`
-  - e.g. run tests: `docker compose run --rm npm run test`
-  - e.g. install a package: `docker compose run --rm npm install some-pacakge`
-- `poetry` commands and tasks (see [`tasks` in `pyproject.toml`](https://github.com/ResonantGeoData/RD-WATCH/blob/phase-ii/django/pyproject.toml#L42)):
-  - e.g. lint and fix code: `docker compose run --rm poetry run task lint:fix`
-  - e.g. build the OpenAPI spec: `docker compose run --rm poetry run task build:openapi`
-  - e.g. install a package: `docker compose run --rm poetry add some-pacakge`
-  - e.g. run tests: `docker compose run --rm poetry run task test`
-
-### Type support for ".vue" imports in VS Code
+## Develop with Docker (recommended quickstart)
+This is the simplest configuration for developers to start with.
+
+1. Run `docker compose up` to start the Django development server and Celery worker, plus all backing services
+   like PostGIS, Redis, RabbitMQ, etc.
+2. Start the client development server:
+   ```sh
+   cd vue
+   npm install
+   npm run dev
+   ```
+3. Access the site, starting at http://localhost:8080/
+4. When finished, use `Ctrl+C`
+
+## Develop Natively (advanced)
+This configuration still uses Docker to run attached services in the background,
+but allows developers to run Python code on their native system.
+
+### Initial Setup
+1. Run `docker compose -f ./docker-compose.yaml up -d`
+2. Install Python 3.10
+3. Install
+   [`psycopg2` build prerequisites](https://www.psycopg.org/docs/install.html#build-prerequisites)
+4. Install Poetry
+5. Run `poetry install`
+6. Run `source ./dev/export-env.sh`
+
+### Run Application
+1.  Ensure `docker compose -f ./docker-compose.yaml up -d` is still active
+2. Run:
+   1. `source ./dev/export-env.sh`
+   2. `poetry run --directory django django-admin runserver`
+3. Run in a separate terminal:
+   1. `source ./dev/export-env.sh`
+   2. `poetry run --directory django celery --app rdwatch.celery worker --loglevel INFO --without-heartbeat`
+4. When finished, run `docker compose stop`
+5. To destroy the stack and start fresh, run `docker compose down -v`
```

Comment:
`-v` will remove/destroy any volumes.  Not sure if you want to destroy the `postgresql-data` and `minio-data` volumes?

Reply:
Added additional instructions clarifying this https://github.com/ResonantGeoData/RD-WATCH/pull/192/commits/3ae6ee62b52631e8ffd99ea8f7392d29b151b87d